src/azul/lambda_layer.py

```diff
@@ -41,18 +41,20 @@ def _update_required(self) -> bool:
             return False
 
     def update_layer(self, force: bool = False):
-        log.info('Using dependency layer package at s3://%s/%s.', config.layer_bucket, self.object_key)
+        log.info('Using dependency layer package at s3://%s/%s.', config.lambda_layer_bucket, self.object_key)
         if force or self._update_required():
             log.info('Staging layer package ...')
             input_zip = self.out_dir / 'deployment.zip'
             layer_zip = self.out_dir / 'layer.zip'
             if force:
                 log.info('Tainting current lambda layer resource to force update')
+                subprocess.run(['make', 'init'], cwd=Path(config.project_root) / 'terraform').check_returncode()
```

Comment:
We should add a `taint_dependency_layer` to the makefile and have it depend on the `init` target. This here is a little brittle.

Reply:
Good call.